staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -194,11 +194,26 @@ func (d *Helper) DeleteOrEvictPods(pods []corev1.Pod) error {
 
 func (d *Helper) evictPods(pods []corev1.Pod, policyGroupVersion string, getPodFn func(namespace, name string) (*corev1.Pod, error)) error {
 	returnCh := make(chan error, 1)
-
+	// 0 timeout means infinite, we use MaxInt64 to represent it.
+	var globalTimeout time.Duration
+	if d.Timeout == 0 {
+		globalTimeout = time.Duration(math.MaxInt64)
+	} else {
+		globalTimeout = d.Timeout
+	}
+	ctx, cancel := context.WithTimeout(context.TODO(), globalTimeout)
```

Comment:
FWIW, while there is no practical difference I am aware of, the golang examples use Background for creating a new context in WithTimeout.

https://golang.org/pkg/context/#example_WithTimeout

Reply:
TODO and background are one in the same.  TODO is just an indication that in the future, we should add support for an actual context to be passed in: https://golang.org/src/context/context.go?s=7599:7618#L203